docs/src/pages/Popper/6-manual-trigger.vue

```diff
@@ -0,0 +1,82 @@
+<title>Manual Triggers</title>
```

Comment:
Somehow this title isn't displayed in the docs. Any clue is appreciated.

Reply:
I think the reason for this is your usage of typescript in this example.

We usually don't use typescript in the documentation examples because the majority of consumers will use single file components without typescript. Since we have two custom webpack loaders that assume to work with plain single file components what you are doing is untested. :D 

If you stop using typescript in this example it will just work.

Reply:
I got it! Thanks for that. I will remove the whole script tag.